db/seed_database.js

```diff
@@ -0,0 +1,85 @@
+var data = require('./seed_data.js');
+var mysql = require('mysql');
+const fs = require('fs');
+const csvParser = require('csv-parse');
+
+const connection = mysql.createConnection({
+  user: 'root',
+  password: 'password',
```

Comment:
Might want to add the password into a separate file and adding that file into your .gitignore for security reasons.